packages/lesswrong/components/spotlights/DismissibleSpotlightItem.tsx

```diff
@@ -6,20 +6,32 @@ import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { useCookiesWithConsent } from '../hooks/useCookiesWithConsent';
 import { HIDE_SPOTLIGHT_ITEM_PREFIX } from '../../lib/cookies/cookies';
 
-export const CurrentSpotlightItem = ({classes}: {
+export const DismissibleSpotlightItem = ({
+  current,
+  spotlight,
+  className,
+  classes,
+}: {
+  current?: boolean,
+  spotlight?: SpotlightDisplay,
+  className?: string,
   classes: ClassesType,
 }) => {
   const { SpotlightItem } = Components
   const { captureEvent } = useTracking()
 
-  const { results: [spotlight] = [] } = useMulti({
+  const { results: currentSpotlightResults } = useMulti({
     collectionName: 'Spotlights',
     fragmentName: 'SpotlightDisplay',
     terms: {
       view: 'mostRecentlyPromotedSpotlights',
       limit: 1
-    }
+    },
+    skip: !current,
   });
+  if (currentSpotlightResults && currentSpotlightResults.length > 0) {
+    spotlight = currentSpotlightResults[0];
+  }
```

Comment:
I'm not a fan of reassigning a prop that is passed in, could you make this a new variable?
```suggestion
const displaySpotlight = currentSpotlightResults?.[0] ?? spotlight;
```